spec/features/stories_manage_spec.rb

```diff
@@ -41,6 +41,7 @@
     accept_confirm do
       click_link "Delete"
     end
+    find("p", text: "You don't have any story yet.")
```

Comment:
@buys-fran Wouldn't it make more sense if it said: "You don't have any stories yet." ?

I know it is unrelated, but I noticed that it sounds off. 😄

Reply:
Oh yeah, for sure. Thank you 👍